schemas/db-schema.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE todo
+    CHARACTER SET = 'utf8'
+    COLLATE = 'utf8_polish_ci';
+
+
+use todo;
+
+
+CREATE TABLE users
+(
+user_id INT(6) NOT NULL PRIMARY KEY,
+username CHAR(30)
```

Comment:
Should probably be VARCHAR
```suggestion
username VARCHAR(30)
```
Also, maybe add an UNIQUE index on this colulmn?